src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -125,9 +125,9 @@ public function start(): bool
          * The part `[a-zA-Z0-9,-]` is related to the PHP ini directive `session.sid_bits_per_character` defined as 6.
          * See https://www.php.net/manual/en/session.configuration.php#ini.session.sid-bits-per-character.
          * Allowed values are integers such as:
-         * - 4 for range `a-f0-9`
+         * - 4 for range `a-f0-9` (@deprecated since Symfony 7.2, only 5 bits per character will be allowed in Symfony 8.0)
          * - 5 for range `a-v0-9`
-         * - 6 for range `a-zA-Z0-9,-`
+         * - 6 for range `a-zA-Z0-9,-` (@deprecated since Symfony 7.2, only 5 bits per character will be allowed in Symfony 8.0)
```

Comment:
```suggestion
         * - 6 for range `a-zA-Z0-9,-` (@deprecated since PHP 8.4)
```

This option is ignored in Symfony 8.0.

Reply:
Symfony will ignore the option in 8.0, no matter the PHP version you're using. As the minimum PHP version required for Symfony 8.0 is still to be chosen, it's safer to keep the message as is.

Reply:
You can still set `session.sid_bits_per_character` to 5 or 6 in `php.ini` and Symfony 8.0 will ignore that also if I understand correctly. Then the part `only 4 bits per character will be allowed in Symfony 8.0` of the comment isn't true either.

Reply:
Indeed, it should be better now